runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/validator/config/AvroValidatorConfigBuilder.java

```diff
@@ -20,16 +20,17 @@
 import java.util.function.Function;
 
 import io.aklivity.zilla.runtime.engine.config.ConfigBuilder;
+import io.aklivity.zilla.runtime.engine.config.ValidatorConfig;
 
 public class AvroValidatorConfigBuilder<T> extends ConfigBuilder<T, AvroValidatorConfigBuilder<T>>
 {
-    private final Function<AvroValidatorConfig, T> mapper;
+    private final Function<ValidatorConfig, T> mapper;
 
     private List<CatalogedConfig> catalogs;
     private String subject;
 
     AvroValidatorConfigBuilder(
-        Function<AvroValidatorConfig, T> mapper)
+        Function<ValidatorConfig, T> mapper)
```

Comment:
I understand why we did this, but it doesn't feel right for the `AvroValidatorConfigBuilder` not to build an `AvroValdiatorConfig` when `Function.identity()` is passed as the `mapper`. Instead it will return `ValidatorConfig` from `.build()`.

Maybe keep this concrete as `AvroValidatorConfig`, and handle the implicit cast in the `AvroValidatorConfig.builder(...)` methods by adjusting the mapper before passing in here?

Reply:
I think it's fixed 🤔 pls chk